docs/product-getting-started-guide/src/main/asciidoc/test-suspend-proc.adoc

```diff
@@ -0,0 +1,46 @@
+[id='testing_suspend-proc']
+= Testing the driver license suspension scenario
+
+The following chapter instructs you how to test the driver license suspension scenario using the data that you specified when you set the Driver License Suspension rules and actions.
+
+[float]
+== Prerequisites
+
+* Created the Driver department traffic violations project
+* Created the Violation and Driver data models
+* Set the Driver License Suspension rules and actions
+
+.Procedure
+. Log in to {CENTRAL}. Click *Menu* -> *Design* -> *Projects*, then *Driver department traffic violations*.
+. Click *Create New Asset* -> *Test Scenario*.
+. In the *Create new Test Scenario window*, enter `Suspend due to total points` in the *Test Scenario* field, and click *+OK*.
+. Click *+GIVEN* to open the *New input* window.
+. Select *Driver* from the *Insert a new fact* pull-down menu.
+. Enter `driver` in the *Fact name* field and click *Add*.
+. Click *Add a field* located under *Driver'[driver]* to open the *Choose a field to add* window.
+. Select *totalPoints* from the *Choose a field to add* pull-down menu and click *+OK*.
+. Click image:6191.png[] next to *totalPoints*, click *Literal value*, then input `10` in the *totalPoints* field.
+. Click *+GIVEN* to open the *New input* window.
+. Select *Violation* from the *Insert a new fact* pull-down menu.
+. Enter `violation` in the *Fact name* field and click *Add*.
+. Click *Add a field* located under *Insert 'Violation'[violation]* to open the *Choose a field to add* window.
+. Select *points* from the *Choose a field to add* pull-down menu and click *+OK*.
+. Click image:6191.png[], then click *Literal value* next to *Literal value*.
+. Input `10` in the *points* field.
```

Comment:
I'd search all "input" instances and use "enter" instead, where used like "input x in the ..." You go back and forth. I'd go with "enter" for sure.

Reply:
done